server/admin_management/api/app/schemas.py

```diff
@@ -47,11 +47,11 @@ class Requester(BaseModel):
     # from datbase and the user record will contain this user_id; and also is the reference-id from other database
     # entities (e.g., app_fam.fam_application_admin, app_fam.fam_access_control_privilege)
     user_id: int
-    user_guid: Optional[Annotated[str, StringConstraints(max_length=32)]] = None
+    user_guid: Annotated[str, StringConstraints(min_length=32, max_length=32)] = None
```

Comment:
I removed the "Optional",  cause we can expect requester and target_user should both have a user_guid when make an api call. And I updated the length of user_guid to be 32, so it won't happen the case that we got an empty string. Also checked the documentation, think the length is 32. 


<img width="831" alt="Screen Shot 2024-05-23 at 4 40 11 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/ff9ae3e5-7202-45f1-9e06-16fea9a3be03">
<img width="836" alt="Screen Shot 2024-05-23 at 4 41 00 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/ddb99beb-cfb5-4278-b026-f38693301e3d">
<img width="837" alt="Screen Shot 2024-05-23 at 4 41 16 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/fe73a231-be92-462f-8c65-ba0351ff4032">